lib/orb/import.ex

```diff
@@ -80,6 +80,12 @@ defmodule Orb.Import do
     end
   end
 
+  defmacro register_memory(global, namespace, min \\ nil) do
```

Comment:
Curious why `register_memory/3` instead of `memory/3`?

Reply:
Just seemed like the thing to do, I was frankly pretty tired when I wrote this so perhaps memory/3 would be more appropriate.